### DISCUSSION TITLE
Tip For Searching old Forum

### DISCUSSION THREAD

**Discussion by ppriede** [ImageMagick/ImageMagick#2636] [General] [answered]
Just a note if you are looking for something in Google and go to the original site (and can't find it)

If you find something like
https://www.imagemagick.org/discourse-server/viewtopic.php?f=1&t=34042
or
https://www.imagemagick.org/discourse-server/viewtopic.php?t=28963

Change the address to
https://**archive**.imagemagick.org/discourse-server/viewtopic.php?f=1&t=34042
https://**archive**.imagemagick.org/discourse-server/viewtopic.php?t=28963

**Comment by urban-warrior** [comment, marked as answer]
We addressed these issues.  Old links from Google should work now.
